src/algorithms/community/label-propagation.jl

```diff
@@ -150,24 +148,24 @@ function compute_vertex_label(h::Hypergraph, v::Int64, vlabels::Dict{Int64,Int64
         return vlabels[v]
     end
 
-    return collect(maxL)[1]
+    return first(maxL)
 end
 
 
 """
-    compute_edge_label(h::Hypergraph, e::Int64, vlabels::Dict{Int64,Int64}, rng::MersenneTwister)
+    compute_edge_label(h::Hypergraph, e::Int64, vlabels::Dict{Int64,Int64}, helabels::Dict{Int64,Int64}, rng::MersenneTwister)
 
 Hyperedges labeling phase. Computes the labels of the hyperedges according  to
 the  most frequent label among the vertices contained in that hyperedge.
 """
-function compute_edge_label(h::Hypergraph, e::Int64, vlabels::Dict{Int64,Int64}, rng::MersenneTwister)
+function compute_edge_label(h::Hypergraph, e::Int64, vlabels::Dict{Int64,Int64}, helabels::Dict{Int64,Int64}, rng::MersenneTwister)
     vₑ = getvertices(h,e)
     eL = Dict{Int64,Int64}()
 
     max = 0
-    maxL = -1
+    maxL = Set{Int64}()
 
-    for v in shuffle!(rng, collect(keys(vₑ)))
+    for v in shuffle!(rng, sort(collect(keys(vₑ))))
```

Comment:
same here - why `sort`?

Reply:
The same comment as above  - we can be deterministic via using an appropiate `Hypergraph` constructor,